common/istio/base/kustomization.yaml

```diff
@@ -6,3 +6,12 @@ resources:
 - gateway.yaml
 transformers:
 - metadataLabelTransformer.yaml
+patchesJson6902:
```

Comment:
Why `patchesJson6902` as opposed to `patches`? Seems like the [former is older](https://github.com/kubernetes-sigs/kustomize/issues/2705#issuecomment-659012281).

Reply:
@rushtehrani No real reason. I didn't realize it was older. Updated.